canonical/demos.haus
# Add logging pages

## Done

- Added logs to the demo cards

## QA

- Check that the demo is running
- Check that opening logs works, for any of the cards
  - Note that the log retrieval can take a few seconds
- Click the `View all logs` button and confirm that the full logs are accessible 

## Issue / Card
[WD-7570](https://warthogs.atlassian.net/browse/WD-7570)

[WD-7570]: https://warthogs.atlassian.net/browse/WD-7570?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
